test/gtest/ucp/ucp_test.cc

```diff
@@ -113,6 +113,18 @@ bool ucp_test::has_transport(const std::string& tl_name) const {
     return check_transport(tl_name, GetParam().transports);
 }
 
+bool
+ucp_test::has_resource(const ucp_test_base::entity *e, const std::string &tl_name) const
+{
+    for (int i = 0; i < e->ucph()->num_tls; ++i) {
```

Comment:
`auto`?

Reply:
actually these changes can be reverted. (no need to refactor this function out of test_wireup)